cypress/integration/show_work_orders.spec.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="cypress" />
+
+import 'cypress-audit/commands'
+
+beforeEach(() => {
+  cy.fixture('work_orders/jobs.json').as('workorderslist')
+  cy.server()
+  cy.visit('/')
+})
+
+describe('Show all work orders', () => {
+  it('Displays work order details, reference, date raised, last update, priority, property and description', () => {
+    //Stub request with work orders response
+
+    cy.route('GET', 'api/work-orders', '@workorderslist')
+    cy.visit('/work-orders/jobs')
+
+    cy.get('.govuk-table').within(() => {
+      cy.contains('th', 'Reference')
+      cy.contains('th', 'Date raised')
+      cy.contains('th', 'Last update')
+      cy.contains('th', 'Priority')
+      cy.contains('th', 'Property')
+      cy.contains('th', 'Description')
+    })
+
+    //can see reference number
+    cy.get('.govuk-table__cell').within(() => {
```

Comment:
Instead of repeating `cy.get('.govuk-table__cell')` for all these examples could you just have all the `cy.contains(...)` in this one block instead? e.g. 
```
cy.get('.govuk-table__cell').within(() => {
  cy.contains(...)
  cy.contains(...)
}
```